test/browser-specific.js

```diff
@@ -388,6 +388,35 @@ describe('browser-specific.js', () => {
       let scores = browserSpecific.scoreBrowserSpecificFailures(runs, expectedBrowsers);
       assert.deepEqual(scores, new Map([['chrome', 0], ['firefox', 0]]));
     });
+
+    it('should handle tests that arent in all (three) browsers', () => {
+      const expectedBrowsers = new Set(['chrome', 'firefox', 'safari']);
+
+      let chromeTree = new TreeBuilder()
+          .addTest('TestA', 'OK')
+          .addSubtest('TestA', 'Test (upper)', 'PASS')
```

Comment:
Is there any significance to the "upper" and "lower" bits in these test names? The test names aren't the same ignoring casing, so it's just the same as calling the tests "Subtest 1" and "Subtest 2"? If so, that would be clearer.

Reply:
See comment below. We need somewhere where `'a'.localeCompare('A')` and `('a' > 'A') - ('a' < 'A')` differ, which just so happens to basically only be case.

Reply:
per discussion with @foolip earlier, gonna change these to `TEST (upper)` and `test (lower)`